Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionProgressReporterListener.cs

```diff
@@ -153,7 +154,14 @@ public static void AddExecutionDetailsToLog(eExecutionPhase objExecutionPhase, s
                     catch (Exception) { }                  
                 }
 
-                Reporter.ToLog(eLogLevel.DEBUG, stringBuilder.ToString());
+                if (WorkSpace.Instance.RunningInExecutionMode || Reporter.ReportAllAlsoToConsole == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4886178)